staging/src/k8s.io/kubectl/pkg/cmd/config/create_cluster.go

```diff
@@ -121,6 +125,9 @@ func (o *createClusterOptions) modifyCluster(existingCluster clientcmdapi.Cluste
 	if o.server.Provided() {
 		modifiedCluster.Server = o.server.Value()
```

Comment:
should specifying a server name clear TLSServerName, similar to how insecureSkipTLSVerify and CA interact?